From faa8dfe0507b56fb8a7666e326177aec7f364071 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Mon, 3 Dec 2012 11:12:08 -0300
Subject: [PATCH] Fix build failure on ppc

ppc/trace.c is using waitstatus bits without including the appropiate
headers, leading to a build failure:

libtool: link:
/home/gustavoz/b/test/output/host/usr/bin/powerpc-buildroot-linux-uclibc-gcc
-Wall -Wsign-compare -Wfloat-equal -Wformat-security -pipe -Os -o ltrace
main.o  ./.libs/libltrace.a -lelf
./.libs/libltrace.a(lt1-trace.o): In function `syscall_p':
trace.c:(.text+0x28): undefined reference to `WIFSTOPPED'
trace.c:(.text+0x40): undefined reference to `WSTOPSIG'

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 sysdeps/linux-gnu/ppc/trace.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/sysdeps/linux-gnu/ppc/trace.c b/sysdeps/linux-gnu/ppc/trace.c
index c152101..4357a1e 100644
--- a/sysdeps/linux-gnu/ppc/trace.c
+++ b/sysdeps/linux-gnu/ppc/trace.c
@@ -29,6 +29,8 @@
 #include <errno.h>
 #include <signal.h>
 #include <string.h>
+#include <sys/types.h>
+#include <sys/wait.h>
 
 #include "backend.h"
 #include "breakpoint.h"
-- 
1.7.8.6

